### PR TITLE
Fix/tech 220 balance breakdown

### PR DESCRIPTION
**Pull request by Ayagoumi**
Changing Balance Breakdown to reflect these new values:

Locked (X) => Deposited (P)
Locked (P) => Bonded (P)
Locked Stakeable (P) => Bonded & Deposited (P)

### CODE REVIEW

File: src/store/modules/assets/assets.ts [hunk 1]
```diff
@@ -597,63 +616,79 @@ const assets_module: Module<AssetsState, RootState> = {
             return amt
         },
 
-        walletPlatformBalanceLocked(state, getters, rootState): BN {
```
Comment (c4t-ag) [inline src/store/modules/assets/assets.ts:600]: Please remove instead commenting out -> github will still remember...

**Review by c4t-ag — DISMISSED** [review]
This PR has to wait until we have re-implemented getBalances API function

File: package.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     },
     "dependencies": {
         "@arcblock/forge-util": "^1.6.3",
-        "@c4tplatform/camino-wallet-sdk": "file:../camino-wallet-sdk/package",
```
Comment (c4t-ag) [inline package.json]: No

**Review by Noctunus — APPROVED** [review]
Code wise it looks fine from my side

**Pull request merged**